app/views/components/_download-link.html.erb

```diff
@@ -0,0 +1,9 @@
+<%
+  download_link_text ||= "Download File"
```

Comment:
Do we need to extra verbosity here, could we just use 'text' since the we know it's a download link as the component is called download link?